.github/workflows/deploy-main.yml

```diff
@@ -1,12 +1,12 @@
-name: Deploy to GitHub Pages
+name: Deployment
```

Comment:
Can you use a naming scheme similar to Oasis Core?
E.g. name this `deploy` or `deployment`?